README.md

```diff
@@ -343,11 +343,6 @@ write(backend, table, dfs, condition="", labels=None, max_in_message=0,
     index_cols=None, partition_keys=None)
 ```
 
-- When the value of the [`iterator`](#method-read-param-iterator) parameter is `False` (default) &mdash; returns a single DataFrame.
```

Comment:
Why did you delete this?

Reply:
I deleted the `iterator` info in the `write` section because it's not relevant to `write` and is already documented for `read` (as indicated in the commit message).